### ISSUE TITLE
"bad format of filter (expected name=value)" on fin up

### ISSUE THREAD

**Issue by lmakarov** [docksal/docksal#111]
```
$ fin up
bad format of filter (expected name=value)
Starting services...
Starting fusion_cli_1
Starting fusion_db_1
Starting fusion_memcached_1
Starting fusion_web_1
```

**Comment by lmakarov** [comment]
This looks to be a docker bug. Reported upstream in https://github.com/docker/docker/issues/30575

**Comment by lmakarov** [comment]
Fixed in docker 1.13.0+
